Flaky TestMultitenantAlertmanager_FirewallShouldBlockHTTPBasedReceiversWhenEnabled

**Describe the bug**

The tested failed here https://github.com/cortexproject/cortex/pull/4137/checks?check_run_id=2457765261 with no relevant changes.

```
--- FAIL: TestMultitenantAlertmanager_FirewallShouldBlockHTTPBasedReceiversWhenEnabled (0.00s)
    --- FAIL: TestMultitenantAlertmanager_FirewallShouldBlockHTTPBasedReceiversWhenEnabled/receiver=webhook_firewall_enabled=false (3.35s)
        multitenant_test.go:452: 
            	Error Trace:	multitenant_test.go:452
            	Error:      	Not equal: 
            	            	expected: true
            	            	actual  : false
            	Test:       	TestMultitenantAlertmanager_FirewallShouldBlockHTTPBasedReceiversWhenEnabled/receiver=webhook_firewall_enabled=false
FAIL
```

--- a/pkg/alertmanager/multitenant_test.go
+++ b/pkg/alertmanager/multitenant_test.go
@@ -292,6 +292,7 @@ func TestMultitenantAlertmanager_FirewallShouldBlockHTTPBasedReceiversWhenEnable
 route:
   receiver: webhook
   group_wait: 0s
+  group_interval: 1s
 
 receivers:
   - name: webhook
@@ -306,6 +307,7 @@ receivers:
 route:
   receiver: pagerduty
   group_wait: 0s
+  group_interval: 1s
 
 receivers:
   - name: pagerduty
@@ -321,6 +323,7 @@ receivers:
 route:
   receiver: slack
   group_wait: 0s
+  group_interval: 1s
 
 receivers:
   - name: slack
@@ -336,6 +339,7 @@ receivers:
 route:
   receiver: opsgenie
   group_wait: 0s
+  group_interval: 1s
 
 receivers:
   - name: opsgenie
@@ -351,6 +355,7 @@ receivers:
 route:
   receiver: wechat
   group_wait: 0s
+  group_interval: 1s
 
 receivers:
   - name: wechat

